### PR TITLE
Add timeout argument to all `requests` calls

### PR DESCRIPTION
**Pull request by droserasprout**
## Description

Fixes bug where some plugin commands hang indefinitely. 8 years have passed since opening https://github.com/psf/requests/issues/3070, and we still have to deal with this stuff 😔 10 seconds value have been chosen as it's used multiple times in the codebase.

## To Do

- [x] ~Documentation. (If you've added a new command-line flag, for example, find the appropriate page under `docs/` to describe it.)~
- [x] Changelog. (Add an entry to `docs/changelog.rst` to the bottom of one of the lists near the top of the document.)
- [x] ~Tests. (Very much encouraged but not strictly required.)~

### CODE REVIEW

File: beetsplug/acousticbrainz.py [hunk 1]
```diff
@@ -142,7 +142,10 @@ def _get_data(self, mbid):
             self._log.debug("fetching URL: {}", url)
 
             try:
-                res = requests.get(url)
+                res = requests.get(
+                    url,
+                    timeout=10,
```
Comment (snejus) [inline beetsplug/acousticbrainz.py:147]: ✍🏼 A nitpick but try to keep this in a single line if it fits. Applies to the rest of the changes :)

**Review by snejus — COMMENTED** [review]
Just a small nitpick about keeping stuff in a single line if it fits

File: beetsplug/acousticbrainz.py (hunk 1 above)
Comment (droserasprout) [reply beetsplug/acousticbrainz.py]: Done, except for couple of calls with 3+ args which become unreadable in a single line.

**Comment by catap** [comment]
Seems like a fix for https://github.com/beetbox/beets/issues/4103

**Review by snejus — APPROVED** [review]
Looks good 👍🏼

**Comment by Serene-Arc** [comment]
lgtm!

**Pull request merged**